package com.sbbots.test.javaQuiz;

public class ExceptionTest{
	
	public Integer divide(int a, int b) {
		
		try{
			return a/b;	
		} finally{
			System.out.println("finally");
		}
		
	}
	public static void main(String[] args) {
		
		ExceptionTest test = new ExceptionTest();
		
		try{
			System.out.println(test.divide(10,0));
		} catch(Exception ex ) {
			System.out.println("Division by 0!");
		}

		
	}
}
